cooked-validators/tests/Cooked/MockChain/BlockChainSpec.hs

```diff
@@ -0,0 +1,39 @@
+module Cooked.MockChain.BlockChainSpec (tests) where
+
+import Cooked.MockChain.BlockChain
+import Cooked.MockChain.Direct
+import qualified Ledger
+import qualified Ledger.Slot as Ledger
+import Test.Tasty
+import Test.Tasty.QuickCheck
+
+tests :: TestTree
+tests =
+  testGroup
+    "BlockChain"
+    [ testGroup
+        "time handling"
+        [ testProperty "borders computed by slotToTimeInterval are included in slot" $
```

Comment:
A tiny nitpick, but probably worth replacing `borders` with `bounds` here and below.